src/homepage/howTo/deleted-message-vault.html

```diff
@@ -0,0 +1,300 @@
+                        WebAdmin rest APIs.
+                    </p>
+                    <p>
+                        It is useful to keep users deleted messages, administrators can helps users to restore their deleted messages in case of delete by accident.
+                        But messages stored in Deleted Message Vault don't exist forever, Deleted Message Vault support to delete messages after given period had been configured
+                        (<i>retentionPeriod</i> property) by the administrators. It also supports to delete a single message by the message id.
+                    </p>
+                    <p>
+                        Deleted Message Vault API allow to search messages by various criteria base on mail properties.
+                        At the moment, these are supported mail properties for searching:
+                    </p>
+                    <ul>
+                        <li>deletion date(ISO-8601 Date String)
+                            <ul>
+                                <li>supports before or equals operator</li>
+                                <li>supports after or equals operator</li>
+                            </ul>
+                        </li>
+                        <li>delivery date(ISO-8601 Date String)
+                            <ul>
+                                <li>supports before or equals operator</li>
+                                <li>supports after or equals operator</li>
+                            </ul>
+                        </li>
+                        <li>recipients(List of string)
+                            <ul>
+                                <li>supports contains operator</li>
+                            </ul>
+                        </li>
+                        <li>sender(String)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                            </ul>
+                        </li>
+                        <li>has attachment(Boolean)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                            </ul>
+                        </li>
+                        <li>subject(String)
+                            <ul>
+                                <li>supports equal matching operator</li>
+                                <li>supports equal ignore case matching operator</li>
+                                <li>supports contains matching operator</li>
+                                <li>supports contains ignore case matching operator (with US locale)</li>
+                            </ul>
+                        </li>
+                    </ul>
+
+                    <header class="major">
+                        <h2><b>Make Deleted Message Vault available in James</b></h2>
```

Comment:
Please give a hands on tutorial starting from the raw dockerCompose...

 - wget xxx
 - Create configuration files
 - Add volumes for modified configuration files
 - Start James

Rationals: I should be able to try this awesome feature out without any James knowledge.